tests/writeAPI/endpointMethods.php

```diff
@@ -0,0 +1,518 @@
+<?php
+/*
+ * Copyright (C) 2020 STFC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ require_once __DIR__ . '/extensionPropertyAbstract.php';
+
+/**
+ * DBUnit test class for endpo methods of Write API
+ *
+ */
+class WriteAPIendpointMethodsTests extends extensionPropertyAbstract {
+  #Some sample data to test with. Case mixing is deliberate - the API should be case insensitive
```

Comment:
If this failed due to case sensitivity magically happening, would it be obvious why?

Reply:
discussed in review. No - but there isn't a simple way of testing this, so this is a quick way of ensuring it is covered